packages/terra-time-input/src/TimeInput.module.scss

```
@@ -105,11 +105,11 @@
 
   .time-input-hour,
   .time-input-minute {
-    background-color: var(--terra-time-input-background, #fff);
-    background-image: var(--terra-time-input-animation-background, none);
+    background-color: var(--terra-time-input-background-color, #fff);
+    background-image: var(--terra-time-input-animation-background-image, none);
```

Let's drop the word `animation`
```suggestion
    background-image: var(--terra-time-input-background-image, none);
```